src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1319,12 +1323,13 @@ public static void writeAdditionalLevelSaveData(WorldData worldData, CompoundTag
         levelTag.put("fml", fmlData);
     }
 
-    public static void readAdditionalLevelSaveData(CompoundTag rootTag)
+    public static void readAdditionalLevelSaveData(LevelStorageSource.LevelDirectory levelDirectory, CompoundTag rootTag)
```

Comment:
This needs bin compatibility as neither this, nor the class is marked as internal.